src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -1345,7 +1340,12 @@ private void DecodeInterpolatedStringHandlerArgumentAttribute(ref DecodeWellKnow
                         return null;
                     }
 
-                    return (BoundInterpolatedStringArgumentPlaceholder.InstanceParameter, null);
+                    return BoundInterpolatedStringArgumentPlaceholder.InstanceParameter;
+                }
+
+                if (extensionParameter?.Name == name)
```

Comment:
>if (extensionParameter?.Name == name) [](http://example.com/codeflow?start=16&length=37)

It looks like for PEParameterSymbol we do this after member's own parameters are checked. Consider doing things in the same order here as well.

Reply:
Actually, checking extension parameter first is consistent what PEParameterSymbol would do for an implementation symbol. So, perhaps, PEParameterSymbol should be adjusted.